app.json

```diff
@@ -57,11 +54,7 @@
       "quantity": 0,
       "size": "Standard-1X"
     },
-    "pulse_listener_pushes": {
-      "quantity": 1,
-      "size": "Standard-1X"
-    },
-    "pulse_listener_tasks": {
+    "pulse_listener": {
```

Comment:
Please have the 3 Pulse listeners listed. Mark the old ones to have 0 dynos while the new one having only 1 dyno.

Reply:
👍